docs/index.md

```diff
@@ -3,12 +3,62 @@
 page_title: "redpanda Provider"
 subcategory: ""
 description: |-
-  Redpanda Data terraform provider
+  The Redpanda Data Terraform provider is used to manage Redpanda Dedicated and Cloud clusters and Kafka resources within them. To use the provider to connect to a Redpanda Cloud cluster, you need to provide a client_id and client_secret.
+  Example Usage
+  This example demonstrates how to use the Redpanda Terraform provider with various fields populated with dummy values. Ensure you replace these values with your actual data.
+  ```hcl
+  provider "redpanda" {
+    clientid      = "yourclientid"
+    clientsecret  = "yourclientsecret"
+    cloud_provider = "AWS" # Supported values: GCP, AWS
+    region         = "us-west-2"
+    zones          = ["us-west-2a", "us-west-2b", "us-west-2c"]
+  }
+  Define a Redpanda cluster resource
+  resource "redpandacluster" "examplecluster" {
+    name           = "example-cluster"
+    cloudprovider = "AWS"
+    region         = "us-west-2"
+    nodecount     = 3
```

Comment:
I don't see this value here: https://github.com/redpanda-data/terraform-provider-redpanda/blob/main/redpanda/models/cluster.go#L23